### PR TITLE
8253667: ProblemList tools/jlink/JLinkReproducible{,3}Test.java on linux-aarch64

### PR DESCRIPTION
**Pull request by dcubed-ojdk**
---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8253667](https://bugs.openjdk.java.net/browse/JDK-8253667): ProblemList tools/jlink/JLinkReproducible{,3}Test.java on linux-aarch64

### Reviewers
 * [Daniel Fuchs](https://openjdk.java.net/census#dfuchs) (@dfuch - **Reviewer**) ⚠️ Review applies to 38997e41f132d20ad6d5dddd53e6f4d2b9d578c2
 * [Igor Ignatyev](https://openjdk.java.net/census#iignatyev) (@iignatev - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/382/head:pull/382`
`$ git checkout pull/382`

### CODE REVIEW

**Comment by dcubed-ojdk** [comment]
@AlanBateman - not sure if 'core-libs' was the right label to add
so I figured I would ping you directly for a review.

**Review by dfuch — APPROVED** [review]

File: test/jdk/ProblemList.txt [hunk 1]
```diff
@@ -856,6 +856,8 @@ sanity/client/SwingSet/src/ScrollPaneDemoTest.java 8225013 linux-all
 
 tools/jlink/JLinkReproducibleTest.java                          8217166 windows-all
 tools/jlink/plugins/CompressorPluginTest.java                   8247407 generic-all
+tools/jlink/JLinkReproducibleTest.java                          8217166 linux-aarch64
```
Comment (iignatev) [inline test/jdk/ProblemList.txt:859]: wouldn't it be better to "join" JLinkReproducibleTest entries?
 ```suggestion
tools/jlink/JLinkReproducibleTest.java                          8217166 windows-all,linux-aarch64
tools/jlink/plugins/CompressorPluginTest.java                   8247407 generic-all
```

**Review by iignatev — APPROVED** [review]

File: test/jdk/ProblemList.txt (hunk 1 above)
Comment (iignatev) [reply test/jdk/ProblemList.txt:859]: actually, you have to join them, as only jtreg honors only the last entry for a test -- [CODETOOLS-7902481](https://bugs.openjdk.java.net/browse/CODETOOLS-7902481)

Comment (dcubed-ojdk) [reply test/jdk/ProblemList.txt:859]: Yes, I should use the existing entry. I didn't think to check for that.

**Comment by yminqi** [comment]
Should we put this for all platforms?

**Comment by dcubed-ojdk** [comment]
Nope. We're currently seeing these failures on linux-aarch64 so that's what started
this ProblemListing adventure...

**Comment by dcubed-ojdk** [comment]
@dfuch and @iignatev - please re-review when you the chance... Thanks!

**Review by iignatev — APPROVED** [review]

**Pull request closed without merging**